openmetadata-service/src/main/java/org/openmetadata/service/resources/dqtests/TestSuiteResource.java

```diff
@@ -125,6 +125,12 @@ public ResultList<TestSuite> list(
               schema = @Schema(type = "string", example = "executable"))
           @QueryParam("testSuiteType")
           String testSuiteType,
+      @Parameter(
+              description = "Include empty test suite in the response.",
```

Comment:
any specific reason @TeddyCr , we should always list the testSuites empty or not if the satisfy other conditions .. no ?